src/module/actor/npc/sheet.ts

```diff
@@ -209,8 +209,13 @@ export class NPCSheetPF2e<TActor extends NPCPF2e> extends CreatureSheetPF2e<TAct
         });
 
         // Adjustments
-        $html.find(".npc-elite-adjustment").on("click", () => this.actor.applyAdjustment("elite"));
-        $html.find(".npc-weak-adjustment").on("click", () => this.actor.applyAdjustment("weak"));
+        $html.find(".adjustment").on("click", (evt) => {
+            const adjustment = String(evt.target.dataset.adjustment);
```

Comment:
```suggestion
        $html.find(".adjustment").on("click", (event) => {
            const adjustment = String(event.target.dataset.adjustment);
```

Reply:
Whoops. Was in a rush so I fell back to old habits!